wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/swervecontrollercommand/RobotContainer.java

```diff
@@ -32,6 +32,7 @@
  * periodic methods (other than the scheduler calls).  Instead, the structure of the robot
  * (including subsystems, commands, and button mappings) should be declared here.
  */
+@SuppressWarnings("removal")
```

Comment:
We should be able to apply this to just the `swerveControllerCommand` variable declaration and not the whole class.